src/main/java/com/google/sps/servlets/UploadOnBlobstoreServlet.java

```diff
@@ -9,7 +9,7 @@
 import javax.servlet.http.HttpServletResponse;
 
 
-@WebServlet("/blobstore-upload-url-gallery")
+@WebServlet("/upload")
```

Comment:
Is this URL naming consistent with the other URL names in the design doc?

Reply:
Yes, we decided that /upload is what we want